deepchecks/tabular/base_checks.py

```diff
@@ -60,6 +61,7 @@ def __init__(self, **kwargs):
     def run(self, dataset, model=None) -> CheckResult:
         """Run check."""
         assert self.context_type is not None
+        send_anonymous_run_event(self)
```

Comment:
why not send here the "kind" instead of searching for it in the path?

Reply:
didn't want to insert logic related to the telemetry in the base checks. also it is a shared code in vision